src/UserService.ts

```diff
@@ -185,6 +190,20 @@ class UserService extends Base {
       },
     );
   }
+
+  async getBillingPageToken({ username }: { username: string }): Promise<BillingPageToken> {
```

Comment:
Why not just username: string? No need for passing the object here

Reply:
Good point! I suppose we are not going to need any other user parameters here. Should be changed now.